SQLite/Core/Connection.swift

```diff
@@ -22,6 +22,7 @@
 // THE SOFTWARE.
 //
 
+import Foundation
```

Comment:
the missing dependency here is NSUUID
(I wonder if it's worth making this explicit, instead of doing these catch-all imports)?

```swift
import Foundation.NSUUID
```